Infrastructure.DataAccess/Infrastructure.DataAccess.csproj

```diff
@@ -1520,8 +1528,14 @@
     <EmbeddedResource Include="Migrations\202209070640375_AddedNoteToItSystemUsageOverviewReadModel.resx">
       <DependentUpon>202209070640375_AddedNoteToItSystemUsageOverviewReadModel.cs</DependentUpon>
     </EmbeddedResource>
-    <EmbeddedResource Include="Migrations\202209091039005_add_itcontract_read_model.resx">
-      <DependentUpon>202209091039005_add_itcontract_read_model.cs</DependentUpon>
+    <EmbeddedResource Include="Migrations\202209090842227_ChangedIsActiveToActiveAccordingToValidityPeriod.resx">
+      <DependentUpon>202209090842227_ChangedIsActiveToActiveAccordingToValidityPeriod.cs</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Migrations\202209130756266_RemovedActiveFromSystemUsage.resx">
+      <DependentUpon>202209130756266_RemovedActiveFromSystemUsage.cs</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Migrations\202209131255106_add_itcontract_read_model.resx">
+      <DependentUpon>202209131255106_add_itcontract_read_model.cs</DependentUpon>
```

Comment:
I had a problem with the "add_itcontract_read_model" migration after merging. When I tried preparing the local DB, it wouldn't detect the old migration, and when I run "add-migration" it created an identical migration to the old one. I tried to fix it, but the only thing that helped was re-adding the migration